packages/tools/devtools/devtools-browser-extension/webpack.test.js

```diff
@@ -46,6 +47,9 @@ module.exports = (env) => {
 			new webpack.ProvidePlugin({
 				process: "process/browser",
 			}),
+			new Dotenv({
+				path: "./.env",
```

Comment:
Make sure you add .env to the gitignore file for this project if it's not already there.

Reply:
Got it. Adding it!